Installing-and-Configuring/OS/index.md

```diff
@@ -0,0 +1,106 @@
+<!-- markdownlint-enable MD033 -->
+
+Heads and Operating Systems
+====
+
+Since a heads only system would be very limited you will need to install at least one Operating System.  The installation procedures vary greatly depending on which distro you choose, which version of that distro, and if you are encrypting partitions.  Some choices you will have to make include:
+
+* partition table
+* LUKS encryption (yes/no)
+* LUKS version (1 or 2)
+* single vs multiboot
+* simple distro vs hypervisor
+
+All together there are a lot of technical decisions.  This document will attempt to inform you enough to make the decisions.
+
+
+partition table
+----
+
+heads currently requires a separate boot partition which is a common configuration for Linux.  The heads model leaves the /boot partition unencrypted.  This is safe because heads will verify files at boot to detect tampering.
+
+When you partition storage make sure to create /boot or have the OS installer do it for you.  Generally, the boot partition is much smaller than the other OS partition(s).  You may keep everything else on 1 partition to make it easier to manage but the kernel and initrd on the boot partition can mount any number of others--encrypted or not.
+
+The other choices mentioned below may also affect the partition layout you choose.
+
+
+root partition encryption
+----
+
+If you have taken the trouble to install and use heads it is likely that you also want to use encryption to protect your data on storage devices.  LUKS is the most commonly used format in a heads based system.  
+
+Since the root partition is the end result of the boot process you may see how encrypting it makes the boot process complicated.  There are multiple schemes for making this setup work and more will probably be invented.  The key issue is deciding where to store the decryption key so that the root partition can be accessed during boot.  Most of the following use cases are out of scope for heads but worth a quick review so that you may make an informed decision.
+
+* key in TPM
+* key encrypted by security dongle (smartcard)
+* key on removable media
+
+For example, the [Pureboot](https://docs.puri.sm/PureBoot.html) model (based on heads) uses a security dongle (aka smartcard) to encrypt the LUKS decryption key and stores it on /boot which is not encrypted.  This key is decrypted on boot by the smartcard protected by a user PIN.  The smartcard is also used to sign the important files in /boot which are verified at each boot by heads.  
+
+Your other partitions may be encrypted or not based on your preferences and this is outside of the scope of heads.
+
+### Using TPM for decryption of drives
+
+Heads can seal a Disk Unlock Key using the TPM.  This will ensure that upon booting only a disk unlock passphrase and the proper PCR state can access the data on the root partition.  This approach does not use a Security Dongle for disk decryption and may be used to lock the encrypted data to the hardware.
```

Comment:
Heads can seal a Disk Unlock Key using [the TPM NV space](https://osresearch.net/Keys/#tpm-disk-encryption-key). 

 This will ensure that upon booting, only a disk unlock passphrase and the proper TPM PCR state will lead to decryption of the data on the root partition, and additional file being passed to the OS for it to use that secret early in boot.  
 
Not true: This approach does not use a Security Dongle for disk decryption and may be used to lock the encrypted data to the hardware.

True: https://osresearch.net/Keys/#tpm-disk-encryption-key

Reply:
Which part is true vs not true? 
* TPM disk decrypt model does or does not use security dongle? for measurements or decrypt?
* TPM disk decrypt model may be used to lock encrypted data to hardware.  true or false?